CONTRIBUTING.md

```diff
@@ -274,6 +275,13 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### Maintaining code
+
+This section is more dedicated to newly added maintainers of the repo, this new status allow you to merge pull request, you'll find multiple tips and links to where look for those requests in need of merging, and the one which need a final approval and review from @cart.
```

Comment:
This sentence is very long :p Can we split this up?

Reply:
I splitted up the sentence in three parts, let me now if it doesn't fit with what you had in mind